src/visual.ts

```diff
@@ -215,88 +243,131 @@ module powerbi.extensibility.visual {
         private static dateMaxCutter: number = .05;
         private static makeSomeSpaceForCounter: number = .10;
         private static converter(dataView: DataView, visualHost: IVisualHost): LineDotChartViewModel {
-            let categorical: LineDotChartColumns<DataViewCategoryColumn & DataViewValueColumn[]> = LineDotChartColumns.getCategoricalColumns(dataView);
+            let categorical: LineDotChartColumns<DataViewCategoryColumn & DataViewValueColumn[]>
+                = LineDotChartColumns.getCategoricalColumns(dataView);
+
             if (!categorical
                 || !categorical.Date
+                || !categorical.Date.source
                 || _.isEmpty(categorical.Date.values)
                 || !categorical.Values
                 || !categorical.Values[0]
+                || !categorical.Values[0].source
                 || _.isEmpty(categorical.Values[0].values)) {
                 return null;
             }
 
-            let categoryType: valueType = AxisHelper.getCategoryValueType(categorical.Date.source, true);
+            const valuesColumn: DataViewValueColumn = categorical.Values[0],
```

Comment:
The ```valuesColumn``` is actually a column which contains values. It means that we have a single column with some values inside.